### PR TITLE
docs(elements): Edge supports Web Components

### LINKED ISSUE #35556: Angular Elements / Custom Components now supported in IE Edge?
# 📚 Docs or angular.io bug report

In https://angular.io/guide/elements#browser-support-for-custom-elements you say IE Edge is working on an implementation to support custom elements. But according to https://caniuse.com/#search=custom%20elements, as of Jan 14, 2020, IE Edge does support custom components.  

See also https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_custom_elements

### PR DESCRIPTION
**Pull request by abraham**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: fixes https://github.com/angular/angular/issues/35556

## What is the new behavior?

Edge is now listed as supporting Web Components

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by gkalpak — CHANGES_REQUESTED** [review]
Thx for submitting a PR, @abraham :+1:
If I am not mistaken, Web Components are only supported in the new Edge which is based on Chromium, but not on the one that is based on EdgeHTML, which is still the default on Windows 10. So, (for the time being) if people don't manually switch to the new Edge, they will not get Web Components supported natively in their Edge, which might confuse them.

I think it would be a good idea to make it clear in our docs that the new (Chromium-based) Edge supports Web Components and the old (EdgeHTML-based) Edge does not.
(At least, until the Chromium-based one becomes the default.)

WDYT?

**Comment by abraham** [comment]
@gkalpak I was thinking that Chromium Edge was the default already. 👍 I've updated it to list as `Edge (Chromium)`. Is it enough that `Edge (EdgeHTML)` would fall under other browsers? Or would you prefer to explicitly list it?

File: aio/content/guide/elements.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 _Angular elements_ are Angular components packaged as _custom elements_ (also called Web Components), a web standard for defining new HTML elements in a framework-agnostic way.
 
-[Custom elements](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_custom_elements) are a Web Platform feature currently supported by Chrome, Firefox, Opera, and Safari, and available in other browsers through polyfills (see [Browser Support](#browser-support)).
+[Custom elements](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_custom_elements) are a Web Platform feature currently supported by Chrome, Edge (Chromium), Firefox, Opera, and Safari, and available in other browsers through polyfills (see [Browser Support](#browser-support)).
```
Comment (gkalpak) [inline aio/content/guide/elements.md]: Nit: I think `Edge (Chromium)` might be a little confusing to people not knowing about the switch. How about `Edge (Chromium-based)` instead?

Comment (abraham) [reply aio/content/guide/elements.md]: done

**Review by gkalpak — APPROVED** [review]
Thx for following up, @abraham! LGTM :+1:

**Comment by gkalpak** [comment]
**merge-assistance:** Global approval for this docs-only change.

**Pull request closed without merging**